qchem/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 <h3>New features since last release</h3>
 
+* The function ``get_spinZ_matrix_elements`` has been added to the
```

Comment:
Will need to be moved into the v0.11 section!

Reply:
Done.